packages/dcos-integration-test/buildinfo.json

```diff
@@ -0,0 +1,7 @@
+{
+  "requires": [
+      "dnspython",
```

Comment:
I was being lazy and didnt write all the shared requirements that are in dcos-image-deps. Come to think of it, the only reason dcos-image-deps was used is that I didnt want to rebuild beautifulsoup4 or add it as a second package. Should I remove dcos-image-deps, grab bs4, and link to all the dependent packages in dcos-image-deps. Or do I leave dcos-image-deps and just add the big reqs for redundancy? (e.g. boto, python-requests, python-retrying, etc...)